test/fixtures/items.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyText
```

Comment:
You can delete the whole `test` directory.  You should setup [RSpec](https://github.com/rspec/rspec-rails) if you want to do any testing.